src/data/defaultAddress.js

```diff
@@ -9,8 +9,8 @@ import { countryCodes } from '@/data/keys/countryCodes';
 export const defaultAddress = {
   [MYPARCEL]: {
     [ADDRESS.CC]: countryCodes.NETHERLANDS,
-    [ADDRESS.NUMBER]: 31,
-    [ADDRESS.POSTAL_CODE]: '2132JE',
+    [ADDRESS.NUMBER]: 68,
```

Comment:
Vanwaar de wijziging naar Paleis Noordeinde?

Reply:
Dat is dus best grappig, ik dacht dat avondlevering niet meer werkte omdat ik die al heel lang niet meer terug had gekregen. Wat blijkt? PostNL weet wat een bedrijfsadres is en daar is geen avondlevering mogelijk "want die zijn 's avonds dicht". En ik wilde ook niet een al te random persoon zijn/haar huis in de defaults zetten, dus dan maar dit :)

Reply:
haha wel tof. 😆